data.sql

```diff
@@ -55,3 +55,38 @@ UPDATE animals SET owner_id = (CASE
                                 WHEN name IN ('Charmander', 'Squirtle', 'Blossom', 'Angemon', 'Boarmon') THEN 4
                                 ELSE 5
                               END);
+
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍